homeassistant/components/tessie/strings.json

```diff
@@ -255,7 +255,9 @@
       "honk": { "name": "Honk horn" },
       "trigger_homelink": { "name": "Homelink" },
       "enable_keyless_driving": { "name": "Keyless driving" },
-      "boombox": { "name": "Play fart" }
+      "boombox": { "name": "Play fart" },
+      "frunk": { "name": "Open frunk" },
+      "trunk": { "name": "Open/Close trunk" }
```

Comment:
Didn't look but this could rename (entity_id) the buttons so does it?

Reply:
Integration is new in 2024.1